crypto/keys/private.go

```diff
@@ -26,15 +28,24 @@ import (
 	"io/ioutil"
 
 	"github.com/google/trillian"
+	"github.com/google/trillian/crypto/keyspb"
 )
 
+// The size of an RSA key generated by this library, in bits, if not overridden.
+const defaultRsaKeySizeInBits = 2048
+
 // SignerFactory creates signers for Trillian trees.
 // A signers may be created by loading a private key, interfacing with a HSM,
 // or sending network requests to a remote key management service, to give a few
 // examples.
 type SignerFactory interface {
```

Comment:
Oh, I missed that. My bad. :/

Yep, a comment or a slight rename might same me from making the same comment again in a few days.